VERSIONING.md

```diff
@@ -0,0 +1,33 @@
+# Versioning
+
+The NMPolicy versioning follows the [semVer specification](https://semver.org/#semantic-versioning-specification-semver)
+with the following meanings for minor, major and patch upgrade levels:
+
+- major: breaking changes at main branch
+- minor: non breaking changes at main branch
+- patch: patch/bugfix changes at release branch
+
+
+## Release bugfix procedure
```

Comment:
I think there is a need to mention that the fix should first be merged to the main branch (when applicable) and then cherry-pick down to the release branches. Patching directly a release branch is usually and exception, in most cases it should flow through the main branch.